src/Document/Entry.php

```diff
@@ -0,0 +1,253 @@
+<?php
```

Comment:
General comment: Why is all the parameter and return types `mixed` in these `src/Document/*` files?

Reply:
Because I just let phpstorm generate the getters and setter, since there is no equivalent of make:entity with Documents. So they are written by hand.
Will fix

Reply:
Fixed